.spelling

```diff
@@ -80,3 +80,7 @@ H.264
 VP8
 codec
 bitrates
+75
+30
+60
+15
```

Comment:
I think it would make more sense to add `ignore-number` (`-n`) option to spell checker

https://github.com/lukeapage/node-markdown-spellcheck?tab=readme-ov-file#ignore-numbers--n---ignore-numbers